tests/test_config.py

```diff
@@ -54,6 +54,7 @@ def test_load(self):
         )
         self.assertEqual(config.mqtt.topic_prefix(123), "brultech-serial2mqtt-123")
         self.assertIsNone(config.mqtt.username)
+        self.assertEqual(config.mqtt.usetls, False)
```

Comment:
Bonus points if you want to add another PR to add some additional tests to this object to make sure it works right :)

Reply:
Will do